LoRaEngine/LoraKeysManagerFacade/ConcentratorCredentialsFunction.cs

```diff
@@ -62,12 +64,13 @@ public class ConcentratorCredentialsFunction
 
         internal async Task<IActionResult> RunFetchConcentratorCredentials(HttpRequest req, CancellationToken cancellationToken)
         {
-            var stationEui = req.Query["StationEui"];
-            if (StringValues.IsNullOrEmpty(stationEui))
+            var rawStationEui = req.Query["StationEui"];
+            if (StringValues.IsNullOrEmpty(rawStationEui))
             {
                 this.logger.LogError("StationEui missing in request");
                 return new BadRequestObjectResult("StationEui missing in request");
             }
+            var stationEui = StationEui.Parse((string)rawStationEui);
```

Comment:
How about just going with `StationEui.TryParse` here?

```suggestion
            if (!StationEui.TryParse((string)req.Query["StationEui"], out var stationEui))
            {
                this.logger.LogError("StationEui missing in request or invalid");
                return new BadRequestObjectResult("StationEui missing in request or invalid");
            }
```